webpack.config.js

```diff
@@ -37,6 +38,7 @@ const plugins = isProduction
     },
   }),
   new webpack.optimize.AggressiveMergingPlugin(),
+  new OfflinePlugin(),
```

Comment:
This is literally all it took to create and register a simple service worker for static asset caching. Pretty sweet.